packages/@lwc/engine-core/src/framework/component.ts

```diff
@@ -14,47 +14,33 @@ import {
 import { VM, scheduleRehydration } from './vm';
 import { VNodes } from '../3rdparty/snabbdom/types';
 import { ReactiveObserver } from '../libs/mutation-tracker';
-import { LightningElementConstructor } from './base-lightning-element';
 import { Template, isUpdatingTemplate, getVMBeingRendered } from './template';
 
-export type ErrorCallback = (error: any, stack: string) => void;
```

Comment:
I suspect no one was using this one, right?

Reply:
This one was used in a [single place](https://github.com/salesforce/lwc/pull/2157/files#diff-733875d4412ded6a6ed2d92c4497bb55b643ece444060df3cefbe6872163ab4bR61). I replaced it with the `LightningElement['errorCallback']` type.